source/main/physics/BeamFactory.cpp

```diff
@@ -164,39 +100,31 @@ ActorManager::ActorManager()
 
     if (RoR::App::app_multithread.GetActive())

```

Comment:
The `if (gEnv->threadpool) {} else {}` checks don't hurt anyone. Besides they are even needed if `app_multithread.GetActive() == true`. Since multi-threading does not imply that the threadpool exists.

> and singlethreaded CPUs are history

Single threaded mode should stay until we can proof that our multi threading is bug free.